documentation/pom.xml

```diff
@@ -89,10 +89,16 @@
       <version>${camel.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>compile</scope>
+      <version>${version.junit5}</version>
+    </dependency>
+
     <dependency>
       <groupId>org.junit.jupiter</groupId>
```

Comment:
Is the API dependency not a transitive of the one below?

Reply:
Yes, because the inherited dependency is in the test scope.

Reply:
The main pom has jupiter and jupiter-api in the test scope.